ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/execution/InsertValuesExecutorTest.java

```diff
@@ -548,7 +554,7 @@ public void shouldFailOnDowncast() {
 
     // Expect:
     expectedException.expect(KsqlException.class);
-    expectedException.expectMessage("Expected type INTEGER for field");
+    expectedException.expectMessage("Expected type Schema{INT32} for field");
```

Comment:
Can we change this to INTEGER in the code? User does not know about INT32/INT64. These are internal types.